.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "t/grpc_server_example"]
 	path = t/grpc_server_example
 	url = https://github.com/api7/grpc_server_example
+[submodule ".github/actions/action-create-comment"]
+	path = .github/actions/action-create-comment
+	url = https://github.com/actions-ecosystem/action-create-comment
```

Comment:
If no comment is currently supported, still need to import this module?

Reply:
rm it